app/src/main/java/com/example/marvelapp/data/service/api/MarvelApi.kt

```diff
@@ -14,5 +13,5 @@ interface MarvelApi {
     fun getCharacter(): Call<MarvelBaseResponse<MarvelResponse<List<CharacterResponse>>>>
 
     @GET("/v1/public/characters/{characterId}")
-    fun getSingleCharacter(@Path("characterId") id: Int): Call<MarvelBaseResponse<MarvelResponse<List<CharacterDetailResponse>>>>
+    fun getSingleCharacter(@Path("characterId") id: Int): Call<MarvelBaseResponse<MarvelResponse<List<CharacterEntity>>>>
```

Comment:
when I read entity, I think you are getting that CharacterEntity from domain. This character is we receive from service, maybe the correct name could be CharacterResponse

Reply:
You had a CharacterResponse, if it is correct use here pls

Reply:
Thx for the advice, now its fixed